dpnp/dpnp_utils/dpnp_utils_statistics.py

```diff
@@ -90,3 +187,62 @@ def _get_2dmin_array(x, dtype):
     c *= 1 / fact if fact != 0 else dpnp.nan
 
     return dpnp.squeeze(c)
+
+
+def dpnp_median(
```

Comment:
Could you please measure the performance against numpy (considering different `axis` values) for both `median` and `nanmedian` functions?
It'd be great to have a current picture here.